composer.json

```diff
@@ -93,14 +92,6 @@
             "url": "https://github.com/goalgorilla/social_kpi_lite.git"
         }
     },
-    "scripts": {
-        "post-install-cmd": [
-            "DrupalComposer\\DrupalScaffold\\Plugin::scaffold"
-        ],
-        "post-update-cmd": [
-            "DrupalComposer\\DrupalScaffold\\Plugin::scaffold"
-        ]
-    },
```

Comment:
@navneet0693 This is replace by https://github.com/goalgorilla/social_template/blob/master/composer.json#L35 

The default install folder for the new scaffold plugin is `web` while we expect `html`.

Reply:
@Kingdutch Aah, this makes it much more clear. I have added this change as well.